### PR TITLE
Name functionality

### PR DESCRIPTION
**Pull request by iva-stoycheva**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -2,21 +2,19 @@
...
+  "proxy": "http://localhost:8086",
   "dependencies": {
     "@mui/material": "^5.10.1",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
     "react-router-dom": "^6.3.0",
     "react-scripts": "5.0.1"
   },
-  "devDependencies": {
-    "prettier": "2.7.1"
```
Comment (SlavenGrowy) [inline package.json:19]: Why did we remove prettier as a dev dependency?

File: package.json [hunk 2]
```diff
@@ -2,21 +2,19 @@
   "name": "tic-tac-toe-frontend",
   "version": "0.1.0",
   "private": true,
-  "scripts": {
-    "start": "react-scripts start",
-    "build": "react-scripts build",
-    "eject": "react-scripts eject",
-    "prettier": "prettier --write ."
-  },
+  "proxy": "http://localhost:8086",
```
Comment (SlavenGrowy) [inline package.json:5]: What does this do?

File: package.json [hunk 3]
```diff
@@ -2,21 +2,19 @@
   "name": "tic-tac-toe-frontend",
   "version": "0.1.0",
   "private": true,
-  "scripts": {
-    "start": "react-scripts start",
-    "build": "react-scripts build",
-    "eject": "react-scripts eject",
-    "prettier": "prettier --write ."
-  },
```
Comment (SlavenGrowy) [inline package.json:10]: Can we move the start script over the "dependencies" property?

**Review by SlavenGrowy — CHANGES_REQUESTED** [review]
Almost there

File: package.json (hunk 2 above)
Comment (iva-stoycheva) [reply package.json:5]: We need it when we use fetch.

Comment (SlavenGrowy) [reply package.json:5]: There are downsides to hardcoding the url in the following manner. But we can leave it like this for now.

File: src/components/Players.js [hunk 4]
```diff
@@ -1,9 +1,30 @@
...
+import { userFetchInterval } from '../constants'
 
 const Players = () => {
+  const [onlineUsers, setOnlineUsers] = useState([])
+
+  useEffect(() => {
+    const interval = setInterval(updateOnlineUsersList, userFetchInterval)
+    return () => {
+      clearInterval(interval)
+    }
```
Comment (SlavenGrowy) [inline src/components/Players.js:12]: 👌

**Review by SlavenGrowy — APPROVED** [review]
Good job! 🙌

**Pull request merged**